main.go

```diff
@@ -14,17 +14,11 @@ func main() {
 	}
 
 	t := types.LoadTypes()
-	var ots []string
-	for _, ot := range t.OperationTypes {
-		if ot == types.MinerRewardOpType || ot == types.UncleRewardOpType {
-			// Optimism does not have miner or uncle reward type ops
-			continue
-		}
-		ots = append(ots, ot)
-	}
-	ots = append(ots, MintOpType)
+	t.OperationTypes = append(types.OperationTypes, types.InvalidOpType)
+	// Optimism does not have miner or uncle reward type ops
 	// TODO(inphi): add BurnOpType
-	t.OperationTypes = ots
+	t.OperationTypes = t.OperationTypes[2:]
```

Comment:
This seems like a style regression... it's always assuming that miner and uncle will be the first two options in the slice. This is why i initially added the nice explicit filtering loop above.

Reply:
gotcha